apps/finance/app/src/components/TransferRow.js

```diff
@@ -48,136 +48,110 @@ class TransferRow extends React.Component {
-                  <ActionLabel>View Transaction</ActionLabel>
-                </ContextMenuItem>
-              </ContextMenu>
-            )}
-            {showCopyTransferMessage && (
-              <ConfirmMessageWrapper>
-                <ConfirmMessage onDone={this.handleConfirmMessageDone}>
-                  Transaction URL copied to clipboard
-                </ConfirmMessage>
-              </ConfirmMessageWrapper>
-            )}
-          </ActionsWrapper>
-        </NoWrapCell>
-      </TableRow>
+      <Viewport>
+        {({ below }) =>
+          below('medium') ? (
+            <TableRow>
+              <StyledTableCell>
+                <Grid>
+                  <div>
+                    <div css="display: flex">
+                      <IdentityBadge
+                        networkType={network.type}
+                        entity={entity}
+                        shorten
+                      />
+                    </div>
+                  </div>
+                  <time dateTime={formattedDate} title={formattedDate}>
+                    {format(date, 'dd MMM yyyy')}
+                  </time>
+                  <TextOverflow style={{ marginTop: '5px' }}>
+                    {reference}
+                  </TextOverflow>
+                  <Amount positive={isIncoming} style={{ marginTop: '5px' }}>
+                    {formattedAmount} {token.symbol}
+                  </Amount>
+                </Grid>
+              </StyledTableCell>
+            </TableRow>
+          ) : (
+            <TableRow>
+              <NoWrapCell>
+                <time dateTime={formattedDate} title={formattedDate}>
+                  {format(date, 'dd/MM/yy')}
+                </time>
+              </NoWrapCell>
+              <NoWrapCell>
+                <IdentityBadge
+                  networkType={network.type}
+                  entity={entity}
+                  shorten={!wideMode}
+                />
+              </NoWrapCell>
+              <NoWrapCell title={reference} style={{ position: 'relative' }}>
+                <TextOverflow
+                  style={{ position: 'absolute', left: '20px', right: '20px' }}
+                >
+                  {reference}
+                </TextOverflow>
+              </NoWrapCell>
+              <NoWrapCell align="right">
+                <Amount positive={isIncoming}>
+                  {formattedAmount} {token.symbol}
+                </Amount>
+              </NoWrapCell>
+              <NoWrapCell>
+                <ActionsWrapper>
+                  {network.etherscanBaseUrl && (
+                    <ContextMenu>
+                      <ContextMenuItem onClick={this.handleViewTransaction}>
+                        <IconTokens />
+                        <ActionLabel>View Transaction</ActionLabel>
+                      </ContextMenuItem>
+                    </ContextMenu>
+                  )}
+                  {showCopyTransferMessage && (
```

Comment:
Noticed you removed the commented out context menu (don't remember why we commented it out in the first place), but should we just remove all of the code related to it?

I find copying the transaction hash to be more useful, and worst case opening the etherscan URL gives you that anyway.

Reply:
Ah yes. I’m not sure why we decided to remove it, maybe for iframe permissions yes. I’m going to remove the code, and we’ll probably have a better solution soon within the transaction popover.